README.md

```diff
@@ -57,6 +57,28 @@ exports.onRouteChange = () => {
 }
 ```
 
+### Customize the scrolling behavior
+```js
+anchorate({ 
+  scroller: function (element) {
+    if (!element) return false
+    element.scrollIntoView({ behavior: 'smooth' })
```

Comment:
Just a side note - someday we may want to refactor custom scrollers support asynchronous callback.

Reply:
You could do that in a non-breaking way by allowing scroller to return a then-able/promise rather than a boolean.  Going the node-callback route, would also work, but be a breaking change.  The former could still be a breaking change though, based on whether any callers are supplying their own scroller and relying on truthy semantics instead of returning booleans.